packages/connect/package.json

```diff
@@ -99,6 +101,7 @@
         "karma-jasmine-async": "^0.0.1",
         "karma-sourcemap-loader": "^0.4.0",
         "karma-webpack": "^5.0.1",
+        "node-libs-browser": "^2.2.1",
```

Comment:
🤔 why ?

Reply:
There is `process` polyfill used in tests, which is not defined anywhere and it's taken from node_modules because some other library in monorepo uses it.